scheduler/pmc-processor-task/src/main/kotlin/ac/uk/ebi/pmc/data/MongoRepository.kt

```diff
@@ -37,4 +46,11 @@ class MongoRepository(
         collection.insertOne(FileDoc(file.name, file.absolutePath, accNo))
         return collection.findOne(Filters.eq("path", file.absolutePath))!!.id
     }
+
+    suspend fun getSubFiles(ids: List<ObjectId>): MutableList<Option<FileDoc>> {
```

Comment:
why a mutable list? I do not see any scenario that we need to update the list of files so better keep it inmutable

Reply:
Done